# This feature was removed in kernel 6.5 and the patch was backported to 6.1.39

diff --git a/8812au-20210629-07ac856293e247347b891c5dbd13f3ab8321132d.org/os_dep/linux/wifi_regd.c b/8812au-20210629-07ac856293e247347b891c5dbd13f3ab8321132d/os_dep/linux/wifi_regd.c
index 81e1dc7..b4b0bcd 100644
--- a/os_dep/linux/wifi_regd.c
+++ b/os_dep/linux/wifi_regd.c
@@ -405,10 +405,6 @@ int rtw_regd_init(struct wiphy *wiphy)
 	wiphy->regulatory_flags &= ~REGULATORY_DISABLE_BEACON_HINTS;
 #endif
 
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 19, 0))
-	wiphy->regulatory_flags |= REGULATORY_IGNORE_STALE_KICKOFF;
-#endif
-
 	return 0;
 }
 #endif /* CONFIG_IOCTL_CFG80211 */
